admin/client/src/components/Preview/Preview.js

```diff
@@ -20,7 +20,11 @@ class Preview extends SilverStripeComponent {
         <div className="toolbar--south">
           <div className="btn-toolbar">
             <button className="btn btn-secondary btn-secondary" type="button"><i className="font-icon-edit"></i> Edit</button>
-            <button type="button" data-container="body" className="btn btn-link btn--options" data-toggle="popover" title="Page actions" data-placement="top" data-content="<a href=''>Add to campaign</a><a href=''>Remove from campaign</a>"><i className="font-icon-dot-3"></i></button>
+            {/* More options button and popover for Alpha 2
```

Comment:
I've deleted the commented out code from this template, but I've kept the HTML as an example in the Popover/Tooltip component which isn't being used anywhere at the moment. 

The Popover/Tooltip component isn't complete but as it isn't used at the moment it hopefully wont hold back this getting merged.